dev-utils/poly-cli/index.js

```
@@ -25,95 +25,43 @@ import { exit } from "process";
-            locales: {
-                en: {
-                    "common.json": () =>
-                        readFileSync(
-                            path.resolve(
-                                __dirname,
-                                "./src/static/templates/locales/en/common.json"
-                            )
-                        ),
-                },
-                de: {
-                    "common.json": () =>
-                        readFileSync(
-                            path.resolve(
-                                __dirname,
-                                "./src/static/templates/locales/de/common.json"
-                            )
-                        ),
-                },
-            },
-            static: {
-                "manifest.json": () => manifestTemplate(feature_name, author),
-                "index.html": () =>
-                    readFileSync(
-                        path.resolve(
-                            __dirname,
-                            "./src/static/templates/index.html"
-                        )
-                    ),
-            },
-        },
-        test: [],
-        "package.json": () =>
-            packageTemplate(
-                feature_name,
-                version,
-                description,
-                "src/index.jsx",
-                author,
-                license
-            ),
-        "README.md": () => readmeTemplate(feature_name, description),
-        "rollup.config.mjs": () =>
-            readFileSync(
-                path.resolve(
-                    __dirname,
-                    "./src/static/templates/rollup.config.mjs"
-                )
-            ),
-    };
-
-    if (existsSync(`./${feature_name}`)) {
-        printErrorMsg("Feature already exists in this folder. Aborting!");
-        return;
-    }
+        if (fs.existsSync(`./${feature_name}`)) {
+            printErrorMsg("Feature already exists in this folder. Aborting!");
```

Nit: You could throw an error here, then it will be handled at line 49.

---

It will essentially exit if setup is not run. But yes, throwing an error will prevent complexity problems later on.